openerp/models.py

```diff
@@ -638,10 +554,26 @@ def _build_model(cls, pool, cr):
         # determine the attributes of the model's class
         ModelClass._build_model_attributes(pool)
 
-        # instantiate the model, and initialize it
+        check_pg_name(ModelClass._table)
+
+        # Transience
+        if ModelClass._transient:
+            ModelClass._transient_check_count = 0
+            ModelClass._transient_max_count = config.get('osv_memory_count_limit')
```

Comment:
This code was in `__init__` and thus act a default value overwritable per model (see [1])
Now, it seems to be always the value from config.

[1] https://github.com/odoo/odoo/blob/master/addons/base_import/models/base_import.py#L60